README.md

```diff
@@ -106,6 +106,43 @@ bundle exec rake
 
 To run the feature specs you will need Chrome installed.
 
+### Running a live console
+
+**Accessing a live console is very risky and should only be done as a last
+resort. This should only be done in pairs, and mutating any live data is
+STRONGLY discouraged.**
+
+The console will be ran inside a container instance and won't be on one of the
+web servers, however it will have access to the database.
+
+#### Through the Azure UI
+
+- Navigate to the container instance resource (eg. `s118d01-app-worker-aci`)
```

Comment:
It would be useful to have a link for this, especially for a developer on dxw first-line support who is unfamiliar with this project.

Reply:
I've added a link to the container instances blade, after that it's just a case of choosing the development/production container instance